nemo/collections/nlp/data/language_modeling/megatron/t5_speechlm_dataset.py

```diff
@@ -191,11 +192,15 @@ def load_data(self, dataset):
                 if self.train_task != 'tts':
```

Comment:
If train_task is None, does that mean it will train on all tasks in the manifest? Or what should train_task be if we want to train it on both ASR and TTS?